api_fhir_r4/converters/subscriptionConverter.py

```diff
@@ -69,7 +69,10 @@ def _build_fhir_criteria(cls, fhir_subscription, imis_subscription):
         criteria = deepcopy(imis_subscription.criteria)
         resource = criteria.pop('resource_type')
         arguments = '&'.join(f'{key}={value}' for key, value in criteria.items())
```

Comment:
Are those criteria used in URL? If that's the case I think `value` should be encoded with `urllib.parse.quote`

Reply:
these `arguments` are only used to display the criteria is URL-like format when GETing the subscriptions. This string is not used for any request.

Reply:
But i will update this to use urllib instead